tests/csapi/rooms_members_local_test.go

```diff
@@ -61,3 +65,51 @@ func TestMembersLocal(t *testing.T) {
 	})
 
 }
+
+// sytest: Can get rooms/{roomId}/members at a given point
+func TestMembersAtAGivenPoint(t *testing.T) {
+	deplyoment := Deploy(t, b.BlueprintOneToOneRoom)
+	defer deplyoment.Destroy(t)
+
+	alice := deplyoment.Client(t, "hs1", "@alice:hs1")
+	bob := deplyoment.Client(t, "hs1", "@bob:hs1")
+
+	roomID := alice.CreateRoom(t, map[string]interface{}{
+		"preset": "public_chat",
+	})
+	alice.SendEventSynced(t, roomID, b.Event{
+		Type: "m.room.message",
+		Content: map[string]interface{}{
+			"msgtype": "m.text",
+			"body":    "Hello World!",
+		},
+	})
+
+	resp, _ := alice.MustSync(t, client.SyncReq{})
+	prevBatch := resp.Get("rooms.join." + client.GjsonEscape(roomID) + ".timeline.prev_batch").Str
+
+	bob.JoinRoom(t, roomID, []string{})
+	alice.MustSyncUntil(t, client.SyncReq{}, client.SyncJoinedTo(bob.UserID, roomID))
+
+	alice.SendEventSynced(t, roomID, b.Event{
+		Type: "m.room.message",
+		Content: map[string]interface{}{
+			"msgtype": "m.text",
+			"body":    "Hello back",
```

Comment:
Slightly surprising this message comes from alice, not Bob? but it matches the sytest :shrug: